CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 - Added support for splines in *3D Viewer* node.
 - Added built-in attributes for Mesh object.
-
+- Added Vertex Group as attribute for Mesh object.
```

Comment:
Keep a new line between contents and headings. Some markdown processors only work with that new line.

Reply:
Okay.